cookbooks/swift/recipes/configs.rb

```diff
@@ -61,16 +71,22 @@
 # services
 
 [
-  "rsync",
+  rsync_name,
   "memcached",
```

Comment:
after initial bring up I noticed my swift wasn't working:

    May 20 18:46:29 saio proxy-server[26260]: All memcached servers error-limited (txn: tx58d2fc7710414e65ae251-006287e205)

... but i was able to get it working

```
[vagrant@saio ~]$ diff /etc/sysconfig/memcached /etc/sysconfig/memcached.orig 
5c5
< OPTIONS="-l 127.0.0.1"
---
> OPTIONS="-l 127.0.0.1,::1"
```

Reply:
oh nice! I'll have to add that. Thanks!